### PR TITLE
Fix daily art

### PR DESCRIPTION
**Pull request by SamDepardieu**
Fix a couple of things:

 - Dockerfile was non optimised and rebuilt everything on simple code changes
 - Slack API was using OAuth method but used Webhooks, so wasn't going to work. It is now only using webhooks.

### CODE REVIEW

File: daily-art/main.py [hunk 1]
```diff
@@ -20,6 +24,9 @@
     exclude_sensitive=True, only_interesting=False, exclude_used=False
 )
 
+logger = logging.getLogger()
```
Comment (aCampello) [inline daily-art/main.py]: proper logging

**Comment by SamDepardieu** [comment]
You don't need to schedule messages, as you run a curl.sh script on a cron :p

**Comment by aCampello** [comment]
> You don't need to schedule messages, as you run a curl.sh script on a cron :p

That's right - scheduling was always supposed to be a temporary solution. Btw, this change breaks `slack_bot_plugin.py`, but as you said we'll probably not need to run it.

**Comment by SamDepardieu** [comment]
Does it 😱

**Comment by SamDepardieu** [comment]
We need a test pipeline now

**Pull request merged**